include/glow/Graph/Graph.h

```diff
@@ -565,6 +565,25 @@ class Function final : public Named {
                                  NodeValue data, NodeValue weights,
                                  NodeValue indices, NodeValue lengths);
 
+  /// Create a node, performing SparseLengthsSum operation, using rowwise
+  /// quantization for the input data. Gathers slices of the outer-most
+  /// dimension of Data indexed by Indices vector, and then accumulates them
+  /// into len(Lengths) entries: first Lengths[0] slices are aggregated to
+  /// Result[0], next Lengths[1] slices are aggregated to Result[1],
+  /// etc. I.e. sum(Lengths) must be equal to len(Indices).
+  RowwiseQuantizedSparseLengthsWeightedSumNode *
+  createRowwiseQuantizedSparseLengthsSum(llvm::StringRef name, Tensor &data,
```

Comment:
Passing a `Tensor` for `data` seems unusual vs. a `NodeValue` or a `Constant`.  I guess it makes sense given its role though, so I don't feel too strongly.

Reply:
Yeah, a `Constant`/`NodeValue` would essentially only be used as a container for `Tensor data`. I also thought it made sense to differentiate it from the other creator functions, to make it more clear that the data passed in is rowwise-quantized.

When we add in importer support for this Node which loads already-quantized `data` we can refactor/add a new creator method that takes a `NodeValue`/`Constant`.